src/Workspaces/Core/Portable/Editing/DeclarationModifiers.cs

```diff
@@ -153,6 +155,11 @@ public DeclarationModifiers WithIsWriteOnly(bool isWriteOnly)
             return new DeclarationModifiers(SetFlag(_modifiers, Modifiers.WriteOnly, isWriteOnly));
         }
 
+        public DeclarationModifiers WithIsRef(bool isRef)
```

Comment:
nit: `WithRef`?

Reply:
I think it's the other way around, `WithAsync` and `WithPartial` should be `WithIsAsync` and `WithIsPartial`, they are public unfortunately...

---
In reply to: [251166882](https://github.com/dotnet/roslyn/pull/32037#discussion_r251166882) [](ancestors = 251166882)

Reply:
`withIsRef` looks weird but consistent with the majority of the similar builder methods from this class, so I left it as is.